### PR TITLE
Fix inactive criteria

### LINKED ISSUE #47: 20. [Bug] Revise los criterios de orden de productos y la clasificación de productos inactivos.
Según este [hilo](https://the-collab-lab.slack.com/archives/C02EXJDHQTZ/p1637154779044800), la solución para ordenar los productos propuestos en esta issue #12 está provocando algunos malentendidos. Existe una nueva propuesta para realizar el cálculo y para ordenar los productos en base a la fecha estimada para la próxima compra. Es necesario realizar los ajustes en nuestra aplicación.

También notamos que al crear un nuevo producto, se clasifica automáticamente como inactivo. Necesitamos corregir este comportamiento para evitar la confusión del usuario.

### PR DESCRIPTION
**Pull request by yanurd**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description

Introduces a new function `getKeyByValue` where it receives an object and a value. In this case the object is TimeFrames and the value is the timeFrame from Firestore, then it'll return the key from that value

## Related Issue

Closes #47 

## Acceptance Criteria

Items with zero purchases should use the color that matches their given timeFrame 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    ✓ | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Testing Steps / QA Criteria

- `git fetch`
- `git checkout fix-inactive-criteria`
-  `npm start`
- Create a new item and check it keeps the color associated to the time frame that was choosen

### CODE REVIEW

File: src/components/add-form/index.js [hunk 1]
```diff
@@ -1,11 +1,6 @@
 import React from 'react';
 import './styles.css';
-
-const TimeFrames = {
-  soon: '7',
-  kindOfSoon: '14',
-  notSoon: '30',
-};
+import { TimeFrames } from '../../utils/timeFrames';
```
Comment (mailliwmogollon) [inline src/components/add-form/index.js:3]: good :)

File: src/utils/nextPurchaseDay.js [hunk 2]
```diff
@@ -27,6 +30,13 @@ export const nextPurchaseDay = (
   const daysSinceLastPurchase = diffBetweenTodayAndDate(lastPurchase);
   const nextEstimatedPurchase = 2 * daysUntilNextPurchase;
 
+  if (numberOfPurchases === 0) {
+    /*
+    This either returns a string with soon, kindOfSoon or notAnySoon
+    that'll depend on the timeframe that comes from Firestore
+    */
+    return getKeyByValue(TimeFrames, timeFrame);
+  }
```
Comment (mailliwmogollon) [inline src/utils/nextPurchaseDay.js:39]: good comment

**Review by mailliwmogollon — APPROVED** [review]
thanks for the effort yan

**Review by IsabellaSoares — APPROVED** [review]

**Review by alxmcr — APPROVED** [review]
Good job! LGTM! 💯

**Pull request merged**